server/routes/refounds.js

```diff
@@ -0,0 +1,34 @@
+const excuteQuery = require('../db/db');
+const users = require('./users');
+
+const getRefoundList = async (request, reply) => {
```

Comment:
typo:
Refund instead of Refound
verify instead varify in the next line

Reply:
Applies to filenames as well: `refounds.js` and `refoundsRoutes.js`.